src/platform_impl/ios/view.rs

```diff
@@ -500,7 +500,9 @@ pub unsafe fn create_window(
             let () = msg_send![uiscreen, setCurrentMode: video_mode.video_mode.screen_mode];
             msg_send![window, setScreen:video_mode.monitor().ui_screen()]
         }
-        Some(Fullscreen::Borderless(ref monitor)) => msg_send![window, setScreen:monitor.ui_screen()],
+        Some(Fullscreen::Borderless(ref monitor)) => {
+            msg_send![window, setScreen:monitor.ui_screen()]
+        }
```

Comment:
The `rustfmt` rules changed in the latest stable release, which is why CI is complaining. Update to the newest stable and re-format and this change, as well as the associated CI error, should go away.